content/docs/rfcs/66/README.md

```diff
@@ -0,0 +1,32 @@
+---
+slug: 66
+title: 66/WAKU2-METADATA
+name: Waku Metadata Protocol
+status: raw
+editor: Alvaro Revuelta <alrevuelta@status.im>
+contributors:
+---
+
+# Metadata Protocol
+
+Waku specifies a req/resp protocol that provides information about the node's medatadata.
+
+
+## Protocol id
+
+`/vac/waku/metadata/1.0.0`
+
+## Request
+
+```proto
+message WakuMetadataRequest {
+}
+```
+
+## Response
+
+```proto
+message WakuMessageResponse {
+  uint64 network_id = 1;
```

Comment:
Do we not foresee possibility to support more than one `network_id` here? If for some reason I want my node to support several test networks I may want a `repeated` here.

Reply:
Would we want the node to support multiple networks? Wouldn't it be more sensible to have a node per network (it would definitely make implementation less complex considering things like RLN)

Reply:
I would say one node, one network. It simplifies many things, reduces complexity, edges cases, etc. Running two networks is as easy as spining up another identical waku node changing the `network-id` flag.